packages/cdai/.storybook/.babelrc

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Do we need a per package storybook?

Reply:
maybe not, but at this stage, for ease of transferring the components over we thought it made sense. When we afterwards merge all of these components into 1 central folder we can resume at having just 1 global storybook